### PR TITLE
UPC CZ (for SK as well) and UPC Romania Fix

### PR DESCRIPTION
**Pull request by arvvoid**
## Please check that your pull request pass this checks:

-  [X] My changes respect the [Kodi add-on development rules](https://kodi.wiki/view/Add-on_rules)
-  [X] My code is both Python 2 and 3 compatible
-  [X] My code follows the code style of this project
-  [X] I made sure there wasn't another Pull Request opened for the same update/change
-  [x] I have successfully tested my changes locally

## Types of changes

-  [X] Bug fix (non-breaking change which fixes an issue)
-  [ ] New feature (non-breaking change which adds functionality)
-  [ ] Breaking change (fix or feature that would cause existing functionality to change)
-  [x] Adding a new region/hbo go version

## Description:

Attempting a blind fix for UPC CZ and SK

Linked to #5

### CODE REVIEW

File: hbogolib/constants.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ class HbogoConstants(object):
     # 0 - operator website login form url, 1 - username field name, 2 - password field name, 3 form payload
     eu_redirect_login = {
         'c55e69f0-2471-46a9-a8b7-24dac54e6eb9': ['https://hbogo.skylink.cz/goauthenticate.aspx?client_id=HBO&redirect_uri=https%3a%2f%2fczapi.hbogo.eu%2foauthskylink%2frequest2.aspx&state=5zveHRYBaocYXvjTxHozRg&scope=HBO&response_type=code', 'txtLogin', 'txtPassword', {"__LASTFOCUS": None, "__EVENTTARGET": "btnSubmit", "__EVENTARGUMENT": None, "__VIEWSTATE": None, "__VIEWSTATEGENERATOR": None, "txtLogin": None, "txtPassword": None}],  # Czech Republic: Skylink + Slovakia: Skylink
-        'f0e09ddb-1286-4ade-bb30-99bf1ade7cff': ['https://service.upc.cz/login/pkmslogin.form', 'username', 'password', {"username": None, "password": None}],  # Czech Republic: UPC CZ + Slovakia: UPC CZ
+        'f0e09ddb-1286-4ade-bb30-99bf1ade7cff': ['https://service.upc.cz/login/pkmslogin.form', 'username', 'password', {"username": None, "password": None, "login-form-type": "pwd"}],  # Czech Republic: UPC CZ + Slovakia: UPC CZ
```
Comment (arvvoid) [inline hbogolib/constants.py]: Codacy Issue found: [line too long (229 > 159 characters)](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4589410)

**Comment by arvvoid** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/plugin.video.hbogoeu/plugin.video.hbogoeu/pullRequest?prid=4589410&bid=15410724)

**Comment by arvvoid** [comment]
Users of UPC CZ SK please test and report back: https://github.com/arvvoid/plugin.video.hbogoeu/archive/UPC_cz_sk_fix.zip

User of UPC Romania please test as well

Provide full debug logs if there is an issue if it works please confirm its working and please specify the operator used.

**Comment by durip** [comment]
I put it on the Czech forum, I guess someone will try ...
https://www.xbmc-kodi.cz/prispevek-novy-plugin-hbo-go-cz?pid=77164#pid77164

File: hbogolib/constants.py [hunk 2]
```diff
@@ -101,11 +101,10 @@ class HbogoConstants(object):
...
-        '41a660dc-ee15-4125-8e92-cdb8c2602c5d': ['https://www.upc.ro/rest/v40/session/start?protocol=oidc&rememberMe=true', 'username', 'password', {"username": None, "credential": None}],  # Romania: UPC Romania
-        '6826b525-04dc-4bb9-ada5-0a8e80a9f55a': ['https://rovodafoneauth.hbogo.eu/authservice/OAuth/AuthorizeResponse', 'UserName', 'Password', {"UserName": None, "Password": None, "__RequestVerificationToken": "O6VnKC1ZOUdyewkDuRsvj9FkYL5z3-pNtL_cFTBZxixuA2z_3TUqj5550M0AoyfA9Mw96XAvoFqvwoj3h6gS0DQqOE_kEkg1CKL-nc5KwbU1", "client_id": "ott-web", "redirect_uri": "https://roapi.hbogo.eu/oauthvodafone/request2.aspx", "state": "03ggvM1EXyqFFxWA7W1y4A", "response_type": "code", "IsAccepted": "true", "login": "Login"}],  # Romania: Vodafone Romania 4GTV+
+        '972706fe-094c-4ea5-ae98-e8c5d907f6a2': ['https://my.telekom.ro/oam/server/auth_cred_submit', 'username', 'password', {"username": None, "password": None}],  # Romania: Telekom Romania (My Account)
+        '41a660dc-ee15-4125-8e92-cdb8c2602c5d': ['https://www.upc.ro/rest/v40/session/start?protocol=oidc&rememberMe=false', 'username', 'credential', {"username": None, "credential": None}],  # Romania: UPC Romania
```
Comment (arvvoid) [inline hbogolib/constants.py]: Codacy Issue found: [line too long (215 > 159 characters)](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4589410)

**Comment by arvvoid** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/plugin.video.hbogoeu/plugin.video.hbogoeu/pullRequest?prid=4589410&bid=15410724)

File: hbogolib/constants.py [hunk 3]
```diff
@@ -101,11 +101,10 @@ class HbogoConstants(object):
...
-        '972706fe-094c-4ea5-ae98-e8c5d907f6a2': ['https://my.telekom.ro/oam/server/auth_cred_submit', 'username', 'password', {"username": None, "password": None}],  # Romania: Telekom Romania
-        '41a660dc-ee15-4125-8e92-cdb8c2602c5d': ['https://www.upc.ro/rest/v40/session/start?protocol=oidc&rememberMe=true', 'username', 'password', {"username": None, "credential": None}],  # Romania: UPC Romania
-        '6826b525-04dc-4bb9-ada5-0a8e80a9f55a': ['https://rovodafoneauth.hbogo.eu/authservice/OAuth/AuthorizeResponse', 'UserName', 'Password', {"UserName": None, "Password": None, "__RequestVerificationToken": "O6VnKC1ZOUdyewkDuRsvj9FkYL5z3-pNtL_cFTBZxixuA2z_3TUqj5550M0AoyfA9Mw96XAvoFqvwoj3h6gS0DQqOE_kEkg1CKL-nc5KwbU1", "client_id": "ott-web", "redirect_uri": "https://roapi.hbogo.eu/oauthvodafone/request2.aspx", "state": "03ggvM1EXyqFFxWA7W1y4A", "response_type": "code", "IsAccepted": "true", "login": "Login"}],  # Romania: Vodafone Romania 4GTV+
+        '972706fe-094c-4ea5-ae98-e8c5d907f6a2': ['https://my.telekom.ro/oam/server/auth_cred_submit', 'username', 'password', {"username": None, "password": None}],  # Romania: Telekom Romania (My Account)
```
Comment (arvvoid) [inline hbogolib/constants.py]: Codacy Issue found: [line too long (205 > 159 characters)](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4589410)

**Comment by xjoska** [comment]
Thank you for your work on this.
Unfortunately, this fix has not solved the issue yet.

Please find attached debug log. If can I do any additional debugs, please let me know.
[kodi2.log](https://github.com/arvvoid/plugin.video.hbogoeu/files/3906137/kodi2.log)

File: hbogolib/constants.py [hunk 4]
```diff
@@ -101,11 +101,10 @@ class HbogoConstants(object):
     # 0 - operator website login form url, 1 - username field name, 2 - password field name, 3 form payload
     eu_redirect_login = {
         'c55e69f0-2471-46a9-a8b7-24dac54e6eb9': ['https://hbogo.skylink.cz/goauthenticate.aspx?client_id=HBO&redirect_uri=https%3a%2f%2fczapi.hbogo.eu%2foauthskylink%2frequest2.aspx&state=5zveHRYBaocYXvjTxHozRg&scope=HBO&response_type=code', 'txtLogin', 'txtPassword', {"__LASTFOCUS": None, "__EVENTTARGET": "btnSubmit", "__EVENTARGUMENT": None, "__VIEWSTATE": None, "__VIEWSTATEGENERATOR": None, "txtLogin": None, "txtPassword": None}],  # Czech Republic: Skylink + Slovakia: Skylink
-        'f0e09ddb-1286-4ade-bb30-99bf1ade7cff': ['https://service.upc.cz/login/pkmslogin.form', 'username', 'password', {"username": None, "password": None}],  # Czech Republic: UPC CZ + Slovakia: UPC CZ
+        'f0e09ddb-1286-4ade-bb30-99bf1ade7cff': ['https://service.upc.cz/login/pkmslogin.form', 'username', 'password', {"login-form-type": "pwd", "username": None, "password": None}],  # Czech Republic: UPC CZ + Slovakia: UPC CZ
```
Comment (arvvoid) [inline hbogolib/constants.py]: Codacy Issue found: [line too long (229 > 159 characters)](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4589410)

**Comment by arvvoid** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/plugin.video.hbogoeu/plugin.video.hbogoeu/pullRequest?prid=4589410&bid=15410724)

File: hbogolib/constants.py [hunk 5]
```diff
@@ -101,11 +101,10 @@ class HbogoConstants(object):
     # 0 - operator website login form url, 1 - username field name, 2 - password field name, 3 form payload
     eu_redirect_login = {
         'c55e69f0-2471-46a9-a8b7-24dac54e6eb9': ['https://hbogo.skylink.cz/goauthenticate.aspx?client_id=HBO&redirect_uri=https%3a%2f%2fczapi.hbogo.eu%2foauthskylink%2frequest2.aspx&state=5zveHRYBaocYXvjTxHozRg&scope=HBO&response_type=code', 'txtLogin', 'txtPassword', {"__LASTFOCUS": None, "__EVENTTARGET": "btnSubmit", "__EVENTARGUMENT": None, "__VIEWSTATE": None, "__VIEWSTATEGENERATOR": None, "txtLogin": None, "txtPassword": None}],  # Czech Republic: Skylink + Slovakia: Skylink
-        'f0e09ddb-1286-4ade-bb30-99bf1ade7cff': ['https://service.upc.cz/login/pkmslogin.form', 'username', 'password', {"username": None, "password": None}],  # Czech Republic: UPC CZ + Slovakia: UPC CZ
+        'f0e09ddb-1286-4ade-bb30-99bf1ade7cff': ['https://service.upc.cz/pkmslogin.form', 'username', 'password', {"login-form-type": "pwd", "username": None, "password": None}],  # Czech Republic: UPC CZ + Slovakia: UPC CZ
```
Comment (arvvoid) [inline hbogolib/constants.py]: Codacy Issue found: [line too long (223 > 159 characters)](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4589410)

**Comment by arvvoid** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4589410&bid=15410724)

**Pull request merged**